doc/cascadia/SettingsSchema.md

```diff
@@ -47,6 +47,7 @@ Properties listed below are specific to each unique profile.
 | `cursorShape` | Optional | String | `bar` | Sets the cursor shape for the profile. Possible values: `"vintage"` ( &#x2583; ), `"bar"` ( &#x2503; ), `"underscore"` ( &#x2581; ), `"filledBox"` ( &#x2588; ), `"emptyBox"` ( &#x25AF; ) |
 | `fontFace` | Optional | String | `Cascadia Mono` | Name of the font face used in the profile. We will try to fallback to Consolas if this can't be found or is invalid. |
 | `fontSize` | Optional | Integer | `12` | Sets the font size. |
+| `fontWeight` | Optional | String | `normal` | Sets the weight (lightness or heaviness of the strokes) for the given font. Possible values: `"thin"`, `"extra-light"`, `"light"`, `"semi-light"`, `"normal"`, `"medium"`, `"semi-bold"`, `"bold"`, `"extra-bold"`, `"black"`, `"extra-black"`, or the corresponding numeric representation of DirectWrite font weight. |
```

Comment:
Also mention that you support the numerical value.

Reply:
wait what where? in the type?

Reply:
You will need to consider how Variable fonts are supported.  CSS's font-weight also supports the numerical value, between 100 and 1000 for the lightest to heaviest font weights.

Reply:
I made sure it's both mentioned in profiles.schema.json and settingsschema.md.

Reply:
Yeah, in the type. And I'd add a comment on the description too.